### PR TITLE
testing/libdiscid: new aport

### PR DESCRIPTION
**Pull request by caseorum**
C library for creating MusicBrainz DiscIDs from audio CDs

### CODE REVIEW

File: testing/libdiscid/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,24 @@
+pkgname="libdiscid"
```
Comment (maxice8) [inline testing/libdiscid/APKBUILD]: missing following header:

 # Maintainer:

**Comment by maxice8** [comment]
Please put url above the description in the commit body

File: testing/libdiscid/APKBUILD (hunk 1 above)
Comment (caseorum) [reply testing/libdiscid/APKBUILD]: Added.

**Comment by caseorum** [comment]
> Please put url above the description in the commit body

Done.

**Comment by maxice8** [comment]
Merged with a few changes

**Pull request closed without merging**